server.coffee

```diff
@@ -27,7 +28,7 @@ loadConfig = (logger) ->
     configWrapper(config)
 
 setCORSHeaders = (req, res, next) ->
-  res.setHeader 'Access-Control-Allow-Origin', '*'
+  res.setHeader 'Access-Control-Allow-Origin', ALLOW_ORIGIN_HEADERS
```

Comment:
It's unfortunate that `ALLOW_ORIGIN_HEADERS` needs to be a global.  Having it work this way is relying on the code setting it's value to be called before `setCORSHeaders`, which is a potential race condition.

Why not make `setCORSHeaders` a function which takes your config value:

```coffeescript
setCORSHeaders = (allowOrigin='*') ->
  (req, res, next) ->
    # ...
```

Where it's used on lines 108 and 110, you could then just pass in the value:

```coffeescript
allowOrigin = loadedConfig.get('httpHeaders.allowOrigin').get() ? '*'

# ...

app.post path, parser, setCORSHeaders(allowOrigin), handlers...
```